docs-source/content/release-notes.md

```diff
@@ -39,6 +39,37 @@ draft: false
 
 #### Operator 3.2.0
 
+##### Features
+
+* The operator's container image is based on Oracle Linux 8.
+* WebLogic Server container images based on Oracle Linux 8 are supported.  
+* [Online updates]({{<relref "/userguide/managing-domains/model-in-image/runtime-updates/_index.md#online-updates">}}) of dynamic configuration changes for Model in Image.
+* Automatic injection of the [WebLogic Monitoring Exporter](https://github.com/oracle/weblogic-monitoring-exporter) as a sidecar container.  
+* [Events]({{< relref "/userguide/managing-domains/domain-events.md" >}}) are generated at important moments in the lifecycle of the operator or a domain.
+* [PodDisruptionBudgets]({{<relref "/userguide/introduction/architecture/_index.md#draining-a-node-and-poddisruptionbudget">}}) are generated for clusters improving the ability to maintain cluster availability during planned node shutdowns and Kubernetes upgrade.
+* Additional scripts to assist with common tasks, such as the scaleCluster.sh script.
+* Support for TCP and UDP on the same channel when the SIP protocol is used.
+
+##### Fix for Bug or Regression
+
+* All fixes included in 3.1.1 through 3.1.4 are included in 3.2.0.
+* Resolved an issue where clustered managed servers would not start when the administration server was not running ([#2093](https://github.com/oracle/weblogic-kubernetes-operator/pull/2093)).
+* Model in Image generated domain home file systems that exceed 1 MB are supported ([#2095](https://github.com/oracle/weblogic-kubernetes-operator/pull/2095)).
+* An event and status update are generated when a cluster can't be scaled past the cluster's maximum size ([#2097](https://github.com/oracle/weblogic-kubernetes-operator/pull/2097)).
+* Improve the operator's ability to recover from failures during its initialization ([#2118](https://github.com/oracle/weblogic-kubernetes-operator/pull/2118)).
+* Improve the ability for scalingAction.sh to discover the latest API version ([#2130](https://github.com/oracle/weblogic-kubernetes-operator/pull/2130)).
+* Resolved an issue where the operator's log would show inappropriate warnings related to missing RBAC permissions ([#2138](https://github.com/oracle/weblogic-kubernetes-operator/pull/2138)).
+* Capture WDT errors related to validating the model ([#2140](https://github.com/oracle/weblogic-kubernetes-operator/pull/2140)).
```

Comment:
Capture  -> Captured

Reply:
Is inappropriate the correct word?  or superfluous closer to describe the behavior?

Reply:
The issue fixed here is that while the operator was checking if it had appropriate permissions in each namespace, it was checking for permissions in the operator's own namespace that would only be needed in a managed namespace. So, the warnings were incorrect or misleading. I'd be okay with either of those words, but the warnings were truly wrong.